package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
   },
   "devDependencies": {
+    "http": "^0.0.0",
```

Comment:
Is this needed?

Reply:
For some reason, yarn added it. But it's a node internal module, so it should not be needed, I'll remove it.